src/gui/rss/rssparser.cpp

```diff
@@ -257,7 +257,13 @@ void RssParser::parseRssArticle(QXmlStreamReader& xml, const QString& feedUrl)
         article["title"] = xml.readElementText().trimmed();
       else if (xml.name() == "enclosure") {
         if (xml.attributes().value("type") == "application/x-bittorrent")
```

Comment:
if the content type is "application/x-bittorrent" can't be a magnet link. can you show me an RSS feed with this problem?